code/game/objects/items/stacks/medical.dm

```diff
@@ -122,8 +135,18 @@
 		//After the do_mob to ensure metabolites have had time to process at least one tick.
 		if(reagent && (C.reagents.get_reagent_amount(/datum/reagent/metabolite/medicine/styptic_powder) || C.reagents.get_reagent_amount(/datum/reagent/metabolite/medicine/silver_sulfadiazine)))
 			to_chat(user, "<span class='warning'>That stuff really hurt! You'll need to wait for the pain to go away before you can apply [src] to your wounds again, maybe someone else can help put it on for you.</span>")
+			C.balloon_alert(user, "You fail to apply [src] to yourself!")
```

Comment:
I reckon you should change "to" -> "on".
Reference line 150, it says: "You apply [src] **on** [M]"

Reply:
I will standardise line 150 to "to" instead of "on" since it works for a wider variety of different names that src can be, while on only works for ointment.